xml/System.Runtime/AmbiguousImplementationException.xml

```diff
@@ -73,11 +85,11 @@
         <Parameter Name="innerException" Type="System.Exception" />
       </Parameters>
       <Docs>
-        <param name="message">To be added.</param>
-        <param name="innerException">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="message">The localized error message string.</param>
```

Comment:
Should this also be "A string that describes the error."?

Reply:
Sure, let's do the same:

```suggestion
        <param name="message">A string that describes the error.</param>
```